lib/wallet/test/unit/Cardano/Wallet/Write/TxSpec.hs

```diff
@@ -143,6 +152,28 @@ spec = do
                 Aeson.Success s -> isPlutusScript s `shouldBe` True
                 Aeson.Error e -> expectationFailure e
 
+    describe "TxOut" $ do
+        describe "computeMinAdaQuantity" $ do
+            it "isBelowMinimumCoinForTxOut (setCoin (result <> delta)) \
+               \ == False (Alonzo)"
```

Comment:
I can't help wondering if these long test name strings wouldn't be better as a comment on a separate, named property. Writing them as strings with line continuation characters makes it quite awkward to edit and format, IMO.

Would it be easy to extract these out into separate, named properties?

Reply:
I'm fond/curious of keeping the property implementation close to the `it` description, and not having to maintain the function name and optional doc comment. So I would prefer to continue seeing how this turns out.